schema/to31.xsl

```diff
@@ -63,6 +63,19 @@
 
   <!-- Additions in attributes.mod -->
 
+  <!-- 
+    Remove double elements that have an above value
+    different than yes, then remove the above attribute
```

Comment:
Should this be "Remove double elements that have an above value of yes, then remove the above attribute altogether"?  If `<double above="yes">` best to remove the element altogether   I think that the xsl works right, but the phrasing of the comment seems backwards.

Reply:
Thanks for finding this! The XSLT didn't work either - it matched the comments, and both are wrong. This should be fixed now. 

While working on #285 I put together a simple Python script to test XSLT changes. Future pull requests should be better tested.

Reply:
Oh!  A Python script that applies XSLT for musicxml to reduce it to a minimum compatible version.  I know of a certain python musicology library that could use this to take 3.1/4.0 to 3.0 (I'm aiming to fully support 3.0 before going further -- my way of saying, "No myke, you don't get to use the tasty 3.1/4.0 tools until you catch up on earlier versions")

I finally got a good lyric model in place that makes working with standard lyric objects simple (one `<text>`, one `<syllabic>`, no `<elision>`) but makes more complex (I call it `composite`) lyrics possible.

Reply:
My script isn't quite that ambitious - it just does 4.0 to 3.1 conversions, rather than optionally chaining them to the other XSLT scripts to go further back. It also hardcodes the musicxml.xsd and to31.xsl locations to avoid passing extra arguments. 

The script uses lxml and was informed by the lxml tutorial on XSLT transforms at https://lxml.de/xpathxslt.html. Here's the main part. The path variables are strings rather than Python paths.

```
from lxml import etree

    def transform_to_31(self):
        schema_doc = etree.parse(self.xsd_path)
        schema = etree.XMLSchema(schema_doc)
        xslt_doc = etree.parse(self.xslt_path)
        doc = etree.parse(self.input_path)
        doc40_ok = schema.validate(doc)
        if not doc40_ok:
            log = schema.error_log
            print(log.last_error)
        else:
            transform = etree.XSLT(xslt_doc)
            tree_31 = transform(doc)
            tree_31.write_output(self.output_path)
            print('MusicXML file transformed to 3.1')
```